r/README.md

```diff
@@ -57,7 +57,7 @@ install_arrow()
 
 Conda users on Linux and macOS can install `arrow` from conda-forge with
 
-    conda install -c conda-forge r-arrow
+    conda install -c conda-forge --strict-channel-priority r-arrow
```

Comment:
The potential advantage of doing it in separate steps like

```
conda config --add channels conda-forge
conda config --set channel_priority strict
```

is that this will be remembered if you afterwards install another r package or update r-arrow. If not, you need to remember to add `-c conda-forge --strict-channel-priority` each time you install something.